bofire/strategies/predictives/botorch.py

```diff
@@ -326,10 +329,11 @@ def _optimize_acqf_continuous(
                 fixed_features_list=fixed_features_list,
                 ic_gen_kwargs=ic_gen_kwargs,
                 ic_generator=ic_generator,
-                options={
-                    "batch_limit": 5 if len(nonlinear_constraints or []) == 0 else 1,
-                    "maxiter": 200,
-                },
+                options=self._get_optimizer_options(),  # type: ignore
```

Comment:
Why `type: ignore`?

Reply:
There is something inconsistent with the type hint of botorch.